scm/driver/gitlab/repo.go

```diff
@@ -102,6 +102,10 @@ func (s *repositoryService) ListV2(ctx context.Context, opts scm.RepoListOptions
 	return convertRepositoryList(out), res, err
 }
 
+func (s *repositoryService) ListNamespace(ctx context.Context, namespace string, opts scm.ListOptions) ([]*scm.Repository, *scm.Response, error) {
+	return nil, nil, scm.ErrNotSupported
```

Comment:
gitlab should have an api to get projects in a group path:
https://docs.gitlab.com/ee/api/groups.html#list-a-groups-projects

Reply:
done